tools/api-inventory/scripts/apply2.py

```
@@ -0,0 +1,58 @@
+      (r"<path:[^>]+>","secret.png"),(r"<[^>]*(pid_value|recid|identifier)>","1001"),
+      (r"<[^>]*activity_id>","A-1"),(r"<[^>]*community_id>","comm1"),(r"<[^>]*(id|Id)>","1"),(r"<[^>]+>","1")]:
+        u=re.sub(pat,val,u)
+    return u
+
+def kind(status, method=None, url=None):
+    if status in ("302","401","403"): return "遮断"
+    if status=="500":
+        return "到達" if (method,url) in byp else "遮断"
+    if status in ("404","405"): return "検証不能"
+    if status in ("000","ERR"): return "無応答"
+    if status and status[0] in "245": return "到達"   # 2xx/400/415/308
```

Ruff (0.16.3)

[error] Multiple statements on one line (colon)

(E701)